lib/cli/commands/subscribeorders.ts

```diff
@@ -18,8 +18,11 @@ export const handler = (argv: Arguments) => {
   ensureConnection(argv, true);
 };
 
+let xud: any;
```

Comment:
Minor nit, but this can be `XudClient` instead of `any`.

Reply:
sure thing